<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CryptoServer</class>
 <widget class="QWidget" name="CryptoServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptoServer</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>0</y>
     <width>581</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>1091</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Option">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_ipChoose">
      <property name="title">
       <string>Lisien port</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_Port">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_controlPort">
            <property name="text">
             <string>control port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_filePort">
            <property name="text">
             <string>file port</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Port">
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_publickConnectKey">
      <property name="title">
       <string>通话私钥文件位置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_publicConnectKey"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_publicConnectKey">
         <item>
          <property name="text">
           <string>RSA_OAEP</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_publicConnectKey">
         <property name="text">
          <string>选择...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_digitalSignatureKey">
      <property name="title">
       <string>数字签名公钥位置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_digitalSignatureKey"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_digitalSignatureKey">
         <item>
          <property name="text">
           <string>ED25519</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_digitalSignatureKey">
         <property name="text">
          <string>选择...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_closeConnection">
      <property name="text">
       <string>关闭与客户端的连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
